index.html

```diff
@@ -23,5 +24,152 @@ <h1 class="playfairFont">José Fernández-Caminero</h1>
+          <li class="greenli">JavaScript</li>
+        </ul>
+  
+        <button class="buttonDeco2">See this project
+          <img src="resources/Union.png" alt="Even cooler (White) Arrow Pointing Right">
+          <img src="resources/Union2.svg" alt="Even cooler (Blue) Arrow Pointing Right">
+        </button>
+      </div>
+
+      <img class="fullWidth" src="resources/yellowLaptop.png" alt="A laptop with a yellow design">
+      <div class="darkBlueBackground projectsPadding">
+        <p class="projectnameStyle">Project name goes here</p>
+  
+        <ul class="greenul">
+          <li class="greenli">HTML/CSS</li>
+          <li class="greenli">Ruby on Rails</li>
+          <li class="greenli">JavaScript</li>
+        </ul>
+  
+        <button class="buttonDeco2">See this project
+          <img src="resources/Union.png" alt="Yet even cooler (White) Arrow Pointing Right">
+          <img src="resources/Union2.svg" alt="Yet even cool (Blue) Arrow Pointing Right">
+        </button>
+      </div>
+
+      <img class="fullWidth" src="resources/grayLaptop.png" alt="A laptop with a gray background">
+      <div class="darkBlueBackground projectsPadding">
+        <p class="projectnameStyle">Project name goes here</p>
+  
+        <ul class="greenul">
+          <li class="greenli">HTML/CSS</li>
+          <li class="greenli">Ruby on Rails</li>
+          <li class="greenli">JavaScript</li>
+        </ul>
+  
+        <button class="buttonDeco2">See this project
+          <img src="resources/Union.png" alt="Another cool (White) Arrow Pointing Right">
+          <img src="resources/Union2.svg" alt="Another cool (Blue) Arrow Pointing Right">
+        </button>
+      </div>
+
+      <img class="fullWidth" src="resources/pinkScreen.png" alt="A laptop with a pink screen">
+      <div class="darkBlueBackground projectsPadding">
+        <p class="projectnameStyle">Project name goes here</p>
+  
+        <ul class="greenul">
+          <li class="greenli">HTML/CSS</li>
+          <li class="greenli">Ruby on Rails</li>
+          <li class="greenli">JavaScript</li>
+        </ul>
+  
+        <button class="buttonDeco2">See this project
+          <img src="resources/Union.png" alt="Another cooler (White) Arrow Pointing Right">
+          <img src="resources/Union2.svg" alt="Another cooler (Blue) Arrow Pointing Right">
+        </button>
+      </div>
+    </div>
+  </section>
+
+  <section class="artDeco2">
+    <h2 class="playfairFont margTop1">About</h2>
+    <p class="aboutP">I can help you build one of those big, complex websites. 
+      
+      More than basic web dev, Full-Stack engineering is about building very rich designs and making effecient, complex databases interact with users; among many other things.</p>
+    <button class="buttonDeco margBottom1">
+      <div class="dflex1">
+        <p class="margRight1">Get my CV</p>
+        <img src="resources/ic_download.svg" alt="Downward arrow">
+      </div>
```

Comment:
- [ ] As required for the project and for good semantics rules, kindly make sure the button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: `Active`, `Hover` and `Pressed` (see design)
![Screenshot from 2023-03-24 17-13-16](https://user-images.githubusercontent.com/83042742/227642747-5ce5dd16-abfd-4033-8fb6-28c069c1e73c.png)

Reply:
Hi @tresorsawasawa Are you sure? This div is wrapped inside a button tag. Which has active, hover, and pressed effects. I had to do it this way in order for the arrow icon to be aligned horizontally. Please, kindly reassess this request. Am I confused here? Thanks!

Reply:
Hi @fdezcaminero 

I am really sorry for this comment review as it is already done. 

Just fix other changes I mentioned and move forward

I really sorry for the mistake